packages/core/src/internal/interfaces/index.ts

```diff
@@ -12,9 +12,7 @@ type ExtraTShirtSizes = 'xs' | 'xxl';
 export type TShirtSizes = BasicTShirtSizes | ExtraTShirtSizes;
 
 // STATUSES
-export type StoplightStatusTypes = 'success' | 'warning' | 'danger';
-
-export type StatusTypes = 'default' | 'info' | StoplightStatusTypes;
+export type StatusTypes = 'neutral' | 'info' | 'success' | 'warning' | 'danger';
```

Comment:
Updated components to be more consistent with status types. Some had default/unknown/neutral describing similar statuses types.